common/src/web/virtualAuthenticator.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```

Comment:
Please can we reuse the file that was used in the `py` version of this feature?

Reply:
I have fixed this.